### ISSUE TITLE
Micromamba on aarch64 having issue with ncurses-6.3 creating new environment

### ISSUE THREAD

**Issue by Corpswalker** [mamba-org/mamba#1859]
Running Arch linux VM on a Mac M1.
I"m not able to create on ArchLinux any environment due to problems with the installation of ncurses and probably pyhton-abi too.
On Mac host, micromamba has no problems creating the same environment. I tested with mamba-forge and could create the new environment.

I tried micromamba on the Mac M1 host too, with success. 

Conda info:
~~~
micromamba info

                                           __
          __  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/


            environment : None (not found)
           env location : -
      user config files : /home/niels/.mambarc
 populated config files : /home/niels/.mambarc
       libmamba version : 0.25.0
     micromamba version : 0.25.1
           curl version : libcurl/7.83.1 OpenSSL/1.1.1q zlib/1.2.12 libssh2/1.10.0 nghttp2/1.47.0
     libarchive version : libarchive 3.3.3 zlib/1.2.11 liblzma/5.2.5 bz2lib/1.0.8 liblz4/1.9.2 libzstd/1.4.5
       virtual packages : __unix=0=0
                          __linux=5.18.1=0
                          __glibc=2.35=0
                          __archspec=1=aarch64
               channels : https://conda.anaconda.org/conda-forge/linux-aarch64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/bioconda/linux-aarch64
                          https://conda.anaconda.org/bioconda/noarch
       base environment : /mnt/psf/Documents/micromamba
               platform : linux-aarch64
~~~

Problematic creation of new Py3.9 environment with micromamba
~~~
micromamba create -n gmdb2gmd python=3.9

                                           __
          __  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/

Found conda-prefix at '/mnt/psf/Documents/micromamba/envs/gmdb2gmd'. Overwrite?: [y/N] y
conda-forge/linux-aarch64                                   Using cache
bioconda/noarch                                             Using cache
conda-forge/noarch                                            No change
bioconda/linux-aarch64                                        No change

Transaction

  Prefix: /mnt/psf/Documents/micromamba/envs/gmdb2gmd

  Updating specs:

   - python=3.9


warning  libmamba Invalid package cache, file '/mnt/psf/Documents/micromamba/pkgs/ncurses-6.3-headf329_1/share/terminfo/e/eterm' has incorrect size
warning  libmamba Invalid package cache, file '/mnt/psf/Documents/micromamba/pkgs/ncurses-6.3-headf329_1/share/terminfo/e/eterm-color' has incorrect size
warning  libmamba Extracted package cache '/mnt/psf/Documents/micromamba/pkgs/python_abi-3.9-2_cp39' has invalid size
warning  libmamba Extracted package cache '/mnt/psf/Documents/micromamba/pkgs/python_abi-3.9-2_cp39' has invalid SHA-256 checksum
warning  libmamba Package tarball '/mnt/psf/Documents/micromamba/pkgs/python_abi-3.9-2_cp39.tar.bz2' is invalid
  Package                    Version  Build               Channel                         Size
────────────────────────────────────────────────────────────────────────────────────────────────
  Install:
────────────────────────────────────────────────────────────────────────────────────────────────

  + _openmp_mutex                4.5  2_gnu               conda-forge/linux-aarch64     Cached
  + bzip2                      1.0.8  hf897c2e_4          conda-forge/linux-aarch64     Cached
  + ca-certificates        2022.6.15  h4fd8a4c_0          conda-forge/linux-aarch64     Cached
  + ld_impl_linux-aarch64     2.36.1  h02ad14f_2          conda-forge/linux-aarch64     Cached
  + libffi                     3.4.2  h3557bc0_5          conda-forge/linux-aarch64     Cached
  + libgcc-ng                 12.1.0  h3242a24_16         conda-forge/linux-aarch64     Cached
  + libgomp                   12.1.0  h3242a24_16         conda-forge/linux-aarch64     Cached
  + libnsl                     2.0.0  hf897c2e_0          conda-forge/linux-aarch64     Cached
  + libsqlite                 3.39.2  hf9034f9_1          conda-forge/linux-aarch64     Cached
  + libuuid                   2.32.1  hf897c2e_1000       conda-forge/linux-aarch64     Cached
  + libzlib                   1.2.12  h4e544f5_2          conda-forge/linux-aarch64     Cached
  + ncurses                      6.3  headf329_1          conda-forge/linux-aarch64     Cached
  + openssl                    3.0.5  h4e544f5_1          conda-forge/linux-aarch64     Cached
  + pip                       22.2.2  pyhd8ed1ab_0        conda-forge/noarch            Cached
  + python                    3.9.13  h5016f1d_0_cpython  conda-forge/linux-aarch64     Cached
  + python_abi                   3.9  2_cp39              conda-forge/linux-aarch64        4kB
  + readline                   8.1.2  h38e3740_0          conda-forge/linux-aarch64     Cached
  + setuptools                64.0.3  py39ha65689a_0      conda-forge/linux-aarch64     Cached
  + sqlite                    3.39.2  h69ca7e5_1          conda-forge/linux-aarch64     Cached
  + tk                        8.6.12  hd8af866_0          conda-forge/linux-aarch64     Cached
  + tzdata                     2022b  h191b570_0          conda-forge/noarch            Cached
  + wheel                     0.37.1  pyhd8ed1ab_0        conda-forge/noarch            Cached
  + xz                         5.2.6  h9cdd2b7_0          conda-forge/linux-aarch64     Cached

  Summary:

  Install: 23 packages

  Total download: 4kB

────────────────────────────────────────────────────────────────────────────────────────────────

Confirm changes: [Y/n] 

Transaction starting
python_abi                                           4.0kB @   7.2kB/s  0.6s
Linking ld_impl_linux-aarch64-2.36.1-h02ad14f_2
Linking ca-certificates-2022.6.15-h4fd8a4c_0
Linking libgomp-12.1.0-h3242a24_16
Linking _openmp_mutex-4.5-2_gnu
Linking libgcc-ng-12.1.0-h3242a24_16
Linking libuuid-2.32.1-hf897c2e_1000
Linking xz-5.2.6-h9cdd2b7_0
Linking openssl-3.0.5-h4e544f5_1
Linking ncurses-6.3-headf329_1
critical libmamba File exists: '/mnt/psf/Documents/micromamba/pkgs/ncurses-6.3-headf329_1/share/terminfo/2/2621a', '/mnt/psf/Documents/micromamba/envs/gmdb2gmd/share/terminfo/2/2621a'
~~~

Working creation with mamba
~~~
mamba create -n gmdb2gmd python=3.9

                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (0.24.0) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████


Looking for: ['python=3.9']

conda-forge/linux-aarch64                            5.6MB @ 675.6kB/s  8.3s
conda-forge/noarch                                   9.0MB @   1.1MB/s  8.4s
Transaction

  Prefix: /home/niels/mambaforge/envs/gmdb2gmd

  Updating specs:

   - python=3.9


  Package                    Version  Build               Channel                         Size
────────────────────────────────────────────────────────────────────────────────────────────────
  Install:
────────────────────────────────────────────────────────────────────────────────────────────────

  + _openmp_mutex                4.5  2_gnu               conda-forge/linux-aarch64     Cached
  + bzip2                      1.0.8  hf897c2e_4          conda-forge/linux-aarch64     Cached
  + ca-certificates        2022.6.15  h4fd8a4c_0          conda-forge/linux-aarch64     Cached
  + ld_impl_linux-aarch64     2.36.1  h02ad14f_2          conda-forge/linux-aarch64     Cached
  + libffi                     3.4.2  h3557bc0_5          conda-forge/linux-aarch64     Cached
  + libgcc-ng                 12.1.0  h3242a24_16         conda-forge/linux-aarch64     Cached
  + libgomp                   12.1.0  h3242a24_16         conda-forge/linux-aarch64     Cached
  + libnsl                     2.0.0  hf897c2e_0          conda-forge/linux-aarch64     Cached
  + libsqlite                 3.39.2  hf9034f9_1          conda-forge/linux-aarch64        1MB
  + libuuid                   2.32.1  hf897c2e_1000       conda-forge/linux-aarch64     Cached
  + libzlib                   1.2.12  h4e544f5_2          conda-forge/linux-aarch64     Cached
  + ncurses                      6.3  headf329_1          conda-forge/linux-aarch64     Cached
  + openssl                    3.0.5  h4e544f5_1          conda-forge/linux-aarch64        3MB
  + pip                       22.2.2  pyhd8ed1ab_0        conda-forge/noarch               2MB
  + python                    3.9.13  h5016f1d_0_cpython  conda-forge/linux-aarch64       13MB
  + python_abi                   3.9  2_cp39              conda-forge/linux-aarch64     Cached
  + readline                   8.1.2  h38e3740_0          conda-forge/linux-aarch64     Cached
  + setuptools                64.0.3  py39ha65689a_0      conda-forge/linux-aarch64        1MB
  + sqlite                    3.39.2  h69ca7e5_1          conda-forge/linux-aarch64        1MB
  + tk                        8.6.12  hd8af866_0          conda-forge/linux-aarch64     Cached
  + tzdata                     2022b  h191b570_0          conda-forge/noarch             121kB
  + wheel                     0.37.1  pyhd8ed1ab_0        conda-forge/noarch            Cached
  + xz                         5.2.6  h9cdd2b7_0          conda-forge/linux-aarch64      441kB

  Summary:

  Install: 23 packages

  Total download: 22MB

────────────────────────────────────────────────────────────────────────────────────────────────

Confirm changes: [Y/n] 
tzdata                                             121.0kB @  23.7kB/s  5.1s
xz                                                 440.6kB @  86.1kB/s  5.1s
sqlite                                               1.0MB @ 199.5kB/s  5.1s
libsqlite                                            1.0MB @ 200.4kB/s  5.1s
openssl                                              2.9MB @ 570.0kB/s  5.1s
pip                                                  1.6MB @ 282.0kB/s  0.4s
setuptools                                           1.4MB @ 253.0kB/s  0.6s
python                                              13.4MB @   1.8MB/s  2.5s
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
~~~

Micromamba on Mac M1
~~~
micromamba create -n gmdb2gmd python=3.9

                                           __
          __  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/

conda-forge/osx-arm64                              @   3.2MB/s  1.1s
pkgs/r/osx-arm64                                   118.0 B @  60.0 B/s  2.0s
pkgs/r/noarch                                        1.3MB @ 637.5kB/s  1.1s
pkgs/main/osx-arm64                                767.5kB @ 325.5kB/s  2.4s
conda-forge/noarch                                 @   2.9MB/s  3.3s
pkgs/main/noarch                                   815.5kB @ 137.3kB/s  6.0s

Transaction

  Prefix: /Users/niels/micromamba/envs/gmdb2gmd

  Updating specs:

   - python=3.9


  Package              Version  Build               Channel                    Size
─────────────────────────────────────────────────────────────────────────────────────
  Install:
─────────────────────────────────────────────────────────────────────────────────────

  + bzip2                1.0.8  h3422bc3_4          conda-forge/osx-arm64     152kB
  + ca-certificates  2022.6.15  h4653dfc_0          conda-forge/osx-arm64     153kB
  + libffi               3.4.2  h3422bc3_5          conda-forge/osx-arm64      39kB
  + libsqlite           3.39.2  h2c9beb0_1          conda-forge/osx-arm64     845kB
  + libzlib             1.2.12  ha287fd2_2          conda-forge/osx-arm64      49kB
  + ncurses                6.3  h07bb92c_1          conda-forge/osx-arm64     948kB
  + openssl              3.0.5  h7aea29f_1          conda-forge/osx-arm64       2MB
  + pip                 22.2.2  pyhd8ed1ab_0        conda-forge/noarch          2MB
  + python              3.9.13  h96fcbfb_0_cpython  conda-forge/osx-arm64      13MB
  + python_abi             3.9  2_cp39              conda-forge/osx-arm64       4kB
  + readline             8.1.2  h46ed386_0          conda-forge/osx-arm64     269kB
  + setuptools          64.0.3  py39h2804cbe_0      conda-forge/osx-arm64       1MB
  + sqlite              3.39.2  h40dfcc0_1          conda-forge/osx-arm64     837kB
  + tk                  8.6.12  he1e0b03_0          conda-forge/osx-arm64       3MB
  + tzdata               2022b  h191b570_0          conda-forge/noarch        121kB
  + wheel               0.37.1  pyhd8ed1ab_0        conda-forge/noarch         32kB
  + xz                   5.2.6  h57fd34a_0          conda-forge/osx-arm64     236kB

  Summary:

  Install: 17 packages

  Total download: 25MB

─────────────────────────────────────────────────────────────────────────────────────

Confirm changes: [Y/n] 

Transaction starting
libzlib                                             49.2kB @ 107.9kB/s  0.5s
libffi                                              39.0kB @  82.7kB/s  0.5s
python_abi                                           4.1kB @   7.1kB/s  0.1s
ca-certificates                                    152.6kB @ 222.2kB/s  0.7s
xz                                                 235.7kB @ 310.1kB/s  0.8s
ncurses                                            947.6kB @ 945.7kB/s  1.0s
wheel                                               32.0kB @  26.6kB/s  0.2s
tzdata                                             121.0kB @  90.5kB/s  0.6s
libsqlite                                          845.0kB @ 517.8kB/s  0.9s
sqlite                                             836.8kB @ 498.1kB/s  1.2s
readline                                           268.9kB @ 125.6kB/s  0.9s
openssl                                              2.4MB @   1.1MB/s  0.6s
setuptools                                           1.4MB @ 557.2kB/s  1.3s
bzip2                                              151.8kB @  57.4kB/s  0.4s
pip                                                  1.6MB @ 589.6kB/s  2.1s
tk                                                   3.4MB @   1.1MB/s  1.4s
python                                              12.8MB @   3.8MB/s  1.2s
Linking libffi-3.4.2-h3422bc3_5
Linking libzlib-1.2.12-ha287fd2_2
Linking ncurses-6.3-h07bb92c_1
Linking xz-5.2.6-h57fd34a_0
Linking ca-certificates-2022.6.15-h4653dfc_0
Linking bzip2-1.0.8-h3422bc3_4
Linking tk-8.6.12-he1e0b03_0
Linking libsqlite-3.39.2-h2c9beb0_1
Linking readline-8.1.2-h46ed386_0
Linking openssl-3.0.5-h7aea29f_1
Linking sqlite-3.39.2-h40dfcc0_1
Linking tzdata-2022b-h191b570_0
Linking python-3.9.13-h96fcbfb_0_cpython
Linking python_abi-3.9-2_cp39
Linking setuptools-64.0.3-py39h2804cbe_0
Linking wheel-0.37.1-pyhd8ed1ab_0
Linking pip-22.2.2-pyhd8ed1ab_0
Transaction finished
~~~

**Comment by jonashaag** [comment]
Can you please fix formatting using backticks

**Comment by Corpswalker** [comment]
Somehow the backticks didn't do the job, triple ~ worked

**Comment by jonashaag** [comment]
Can you please try with 0.25.x and with a fresh VM (or clean the pkg cache)

Also are you able to download repodata.json or the packages correctly using only curl?

**Comment by Corpswalker** [comment]
Both micromambas are 0.25.1, the mamba-forge latest version is 0.24.0. VM was freshly created and I cleaned several times micromamba's cache during my attempts.

Today the instantiation **works without errors**, I think the cause was due to geolocation and my inexperience with Mac and Parallels. Yesterday I kept getting consistent bad downloads from ncurses and randomly 1-2 other packages (only ncurses was blocking steadily after every new download).

Today I set up my arch linux guest to use host's VPN, yesterday I was apparently downloading from inside China. 
The package ncurses poses no troubles anymore.  I think the transmission was getting obstructed a little by the "Great Firewall"
